trio/_core/_multierror.py

```diff
@@ -381,10 +381,13 @@ def traceback_exception_init(
     limit=None,
     lookup_lines=True,
     capture_locals=False,
+    compact=False,
```

Comment:
I'm a bit concerned about this introducing some confusion for other projects that are a bit to sensitive about stuff they shouldn't be or who test with trio around and are then confused when something seemingly stdlib is suddenly different without trio.

Reply:
I wouldn't worry about it. It won't affect static analysis tools because they don't understand monkeypatching. The only case I can think of that it might affect is people using `help()` interactively in the REPL, and I don't think that's substantial enough to be worth worrying about.